podvm/Dockerfile.podvm_binaries

```diff
@@ -0,0 +1,44 @@
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Builds pod vm image inside container
+#
+ARG BUILDER_IMG="quay.io/confidential-containers/podvm-builder-ubuntu"
+
+FROM ${BUILDER_IMG} AS podvm_builder
+
+ARG CLOUD_PROVIDER
+ARG PODVM_DISTRO=ubuntu
+ARG AA_KBC=offline_fs_kbc
+# If not provided, uses system architecture
+ARG ARCH=x86_64
+ARG CAA_SRC=""
+ARG CAA_SRC_REF=""
+
+ENV CLOUD_PROVIDER ${CLOUD_PROVIDER}
+ENV PODVM_DISTRO ${PODVM_DISTRO}
+ENV AA_KBC ${AA_KBC}
+ENV ARCH ${ARCH}
+
+RUN if [ -n "${CAA_SRC}" ]; then \
+      rm -rf cloud-api-adaptor && \
+      git clone ${CAA_SRC} cloud-api-adaptor;\
+    fi && \
+    if [ -n "${CAA_SRC_REF}" ]; then \
+      cd cloud-api-adaptor && \
+      git fetch origin ${CAA_SRC_REF} && \
+      git checkout FETCH_HEAD -b ${CAA_SRC_REF} ;\
+    fi
+# Installs add-ons for foreign target, if required
+RUN ./cloud-api-adaptor/podvm/hack/cross-build-extras.sh
+
+RUN cd cloud-api-adaptor/podvm && \
+     make binaries
+
+RUN tar czvf /podvm-binaries.tar.gz -C /src/cloud-api-adaptor/podvm/files usr/
+RUN tar czvf /pause-bundle.tar.gz -C /src/cloud-api-adaptor/podvm/files pause_bundle/
+
+FROM ubuntu:20.04
```

Comment:
I am wondering if the distro specific build stage is necessary ? Can we use the minimal `FROM scratch` instead since we are only copying ?

Reply:
I explicitly added it to make it easier for running commands using the same image if needed. Let's keep it for sometime and we can change it going forward.

Reply:
Sounds good. The advantage of scratch is it's a no-op. So, you actually save some cycles by avoiding creating a layer.